net/dnsviz/Portfile

```diff
@@ -20,11 +20,14 @@ long_description    DNSViz is a tool suite for measurement, diagnostic, and \
                     visualization of Domain Name System (DNS) behavior, \
                     including its security extensions (DNSSEC).
 
-checksums           rmd160 d996637d61539e778cf4035f5c7afefcc03a47ad \
-                    sha256 6f38f3d71b2b9ca3f4cffb003c828574d0c413bcc5112bb52921fa9db4e69259 \
-                    size 388653
+checksums           rmd160 f48389ae74f642a3b25a0690c9760bffcfa0c236 \
+                    sha256 8e2c4d0636296acf704f7eca1ca8fea98b022c920c5517b39dfdc982ce685cd3 \
+                    size 404126
 
-python.default_version  38
+
+python.pep517           no
```

Comment:
why can it not use the default, which is yes for Python 3.7+?

Reply:
It is using installation options that only work with `python.pep517 no`.  In particular, with `python.pep517 yes` (the default for Python 3.7+), the man pages for the package don't get installed to the right location.  The other installation paths are probably less consequential, but still less than ideal.

Reply:
Are we okay with this?

Reply:
sorry for the delay - let me take a look at this tonight. Ideally, we don't want to set it to "no" bit follow the default. Not sure exactly what the issue is you're seeing but it's likely an upstream issue that should/could be fixed.

Reply:
Here is the difference:

With `python.pep517 no`, the following options are used:
```
destroot.destdir    --prefix=${python.prefix} \
                    --root=${destroot} \
                    --install-data=${prefix} \
                    --install-scripts=${prefix}/bin/
```

And here are the results:
```
$ find /opt/local -name '*dnsviz*' 2> /dev/null | grep man
/opt/local/share/man/man1/dnsviz-query.1.gz
/opt/local/share/man/man1/dnsviz-grok.1.gz
/opt/local/share/man/man1/dnsviz-print.1.gz
/opt/local/share/man/man1/dnsviz-probe.1.gz
/opt/local/share/man/man1/dnsviz-graph.1.gz
/opt/local/share/man/man1/dnsviz.1.gz
```
and:
```
$ find /opt/local -name '*dnsviz*' 2> /dev/null | grep bin
/opt/local/bin/dnsviz
```
Because the scripts and the man pages are installed to system-wide locations, I can run `dnsviz` with default PATH and `man dnsviz` with default `MANPATH`.  Everything works great.

However, with the default (effectively `python.pep517 yes`), the following options are used:

```
destroot.destdir    --prefix=${python.prefix} \
                    --destdir=${destroot}
```
(There are no additional options for `python -m installer`, which is used in this case.)

Here are the results, post-installation:
```
$ find /opt/local -name '*dnsviz*' 2> /dev/null | grep man
/opt/local/Library/Frameworks/Python.framework/Versions/3.12/share/man/man1/dnsviz-probe.1
/opt/local/Library/Frameworks/Python.framework/Versions/3.12/share/man/man1/dnsviz-graph.1
/opt/local/Library/Frameworks/Python.framework/Versions/3.12/share/man/man1/dnsviz-query.1
/opt/local/Library/Frameworks/Python.framework/Versions/3.12/share/man/man1/dnsviz-grok.1
/opt/local/Library/Frameworks/Python.framework/Versions/3.12/share/man/man1/dnsviz-print.1
/opt/local/Library/Frameworks/Python.framework/Versions/3.12/share/man/man1/dnsviz.1
```
and:
```
$ find /opt/local -name '*dnsviz*' 2> /dev/null | grep bin
/opt/local/Library/Frameworks/Python.framework/Versions/3.12/bin/dnsviz
```
In this case, the scripts and man pages are installed to non-system-wide locations.  Thus, I can neither run `dnsviz` with default PATH, nor `man dnsviz` with default `MANPATH`.

Reply:
thanks @cdeccio - I will merge this for now. There has to be a way around this but I also couldn't figure it out quickly... let's leave that change to default to `python.pep517` till next time ;)